recipes/nb_anacondacloud/run_test.bat

```diff
@@ -0,0 +1,10 @@
+REM seems to find _build, still
+rmdir /s /q "%PREFIX%\share\jupyter\kernels"
+
+cd /D "%SRC_DIR%"
+
+CALL npm install || EXIT /B 1

```

Comment:
Right, just the tests

On 07:43, Wed, Jun 8, 2016 Mike Sarahan <notifications@github.com> wrote:

> In recipes/nb_anacondacloud/run_test.bat
> <https://github.com/conda-forge/staged-recipes/pull/723#discussion_r66247539>
> :
>
> > @@ -0,0 +1,10 @@
> > +REM seems to find _build, still
> > +rmdir /s /q "%PREFIX%\share\jupyter\kernels"
> > +
> > +cd /D "%SRC_DIR%"
> > +
> > +CALL npm install || EXIT /B 1
>
> Do you need npm install anywhere other than your tests?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/conda-forge/staged-recipes/pull/723/files/f28d3aef9f5908a486bbff2cf911e86b8a29b428#r66247539>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACxRB8M1X7aib9BFWL2EHoYAZHdtN7Kks5qJrkPgaJpZM4IvMk2>
> .
>